build.sbt

```diff
@@ -12,6 +12,12 @@ addCommandAlias("fix", "all compile:scalafix test:scalafix")
 
 resolvers += Resolver.bintrayRepo("hmrc", "releases")
 
+resolvers += "HMRC-open-artefacts-maven2" at "https://open.artefacts.tax.service.gov.uk/maven2"
+
+libraryDependencies ++= Seq(
```

Comment:
Why we are adding it here , when it's there in Appdependencies

Reply:
I have removed this now